WooCommerce/Classes/ViewRelated/Dashboard/Analytics Hub/AnalyticsHubView.swift

```diff
@@ -19,23 +19,20 @@ final class AnalyticsHubHostingViewController: UIHostingController<AnalyticsHubV
 /// Main Analytics Hub View
 ///
 struct AnalyticsHubView: View {
-
     @StateObject var viewModel: AnalyticsHubViewModel
 
     var body: some View {
         ScrollView {
             VStack(alignment: .leading, spacing: Layout.vertialSpacing) {
                 VStack(spacing: Layout.dividerSpacing) {
                     Divider()
-                    Text("Placeholder for Time Range Selection")
-                        .padding(.leading)
-                        .frame(maxWidth: .infinity, minHeight: 84, alignment: .leading)
-                        .background(Color(uiColor: .listForeground))
+
+                    AnalyticsTimeRangeCard(viewModel: viewModel.timeRangeCard)
+                    .background(Color(uiColor: .listForeground))
```

Comment:
Nit: Missing indentation!

```suggestion
                        .background(Color(uiColor: .listForeground))
```

Reply:
Oh, Indeed. I went ahead and adjusted for the other card's declaration in the file too! Fixed here: https://github.com/woocommerce/woocommerce-ios/pull/8159/commits/25799eadbb7ad2ce445f0712b796043ed6473b06